go/logger/standard.go

```diff
@@ -350,3 +349,15 @@ func PickFirstError(errors ...error) error {
 func (log *Standard) SetExternalHandler(handler ExternalHandler) {
 	log.externalHandler = handler
 }
+
+type UnforwardedLogger Standard
+
+func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
+	return (*UnforwardedLogger)(log)
+}
+
+func (log *UnforwardedLogger) Debug(s string, args ...interface{})   { log.internal.Debug(s, args...) }
+func (log *UnforwardedLogger) Error(s string, args ...interface{})   { log.internal.Error(s, args...) }
+func (log *UnforwardedLogger) Warning(s string, args ...interface{}) { log.internal.Warning(s, args...) }
+func (log *UnforwardedLogger) Info(s string, args ...interface{})    { log.internal.Info(s, args...) }
+func (log *UnforwardedLogger) Profile(s string, args ...interface{}) { log.internal.Debug(s, args...) }
```

Comment:
We might want to override the `Critical` and `Fatal` variants?

An easier approach overall might just be to have `GetUnforwardedLogger` just copy the existing logger and then null out the `externalHandler` member. We could override `SetExternalHandler` to panic too, for good measure.